webapp/components/signup/components/signup_email.jsx

```diff
@@ -40,7 +40,7 @@ export default class SignupEmail extends React.Component {
         this.state = this.getInviteInfo();
     }
 
-    componentDidMount() {
+    static componentDidMount() {
```

Comment:
Does this actually work? It seems weird to me to make an "overridden" method static, but then again, it is javascript...